src/explorer/LeetCodeTreeDataProvider.ts

```diff
@@ -197,6 +230,18 @@ export class LeetCodeTreeDataProvider implements vscode.TreeDataProvider<LeetCod
         ].join(os.EOL);
     }
 
+    private updateTreeDataByProblem(problem: IProblem): void {
```

Comment:
The method name her is confusing. Cuz we only handle `isFavorite` here, but the method name looks like we can update all the fields.

Reply:
I thought we might handle the case of updating other `treeData` fields the at a later time. Seems a overdesign here?
Should I leave a comment announcing `modification of other categories is not needed yet` here, or just change the method name?

Reply:
Make sure now other fields will be updated here in the near PRs, e.g. when a problem is accepted, there will be some code here to update other fields.